operator/controllers/kyma_controller.go

```diff
@@ -279,6 +303,15 @@ func (r *KymaReconciler) HandleProcessingState(ctx context.Context, kyma *v1alph
 
 func (r *KymaReconciler) HandleDeletingState(ctx context.Context, kyma *v1alpha1.Kyma) (bool, error) {
 	logger := log.FromContext(ctx)
+	if r.EnableKcpWatcher {
+		if err := r.RemoveWebhookChart(ctx, kyma, r.RemoteClientCache, r.Client); client.IgnoreNotFound(err) != nil {
+			return false, fmt.Errorf(": %w", err)
+		}
+
+		if chartRemoved := r.IsSkrChartRemoved(ctx, kyma, r.RemoteClientCache, r.Client); !chartRemoved {
+			return true, nil
```

Comment:
if removed, don't return, otherwise following deletion will not processing, and please add a logger here

Reply:
Here, I am returning when chart is not removed. I have refactored it to be more obvious 😄 
Sure, I added a debug log before returning 👍